packages/vuetify/src/unplugin/unimport.ts

```diff
@@ -0,0 +1,117 @@
+// Types
+import type { Preset, PresetImport } from 'unimport'
+import type { DirectiveName } from './types'
+
+export interface VuetifyComposablesOptions {
+  /**
+   * Prefix Vuetify composables:
+   * - when prefix set to `true` will use `useV`: `useVDate`.
+   * - when prefix is a string will use `use<prefix>`: `useVuetifyDate` with `prefix: 'Vuetify'`.
+   */
+  prefix?: true | string
+}
+export interface VuetifyOneComposablesOptions {
+  /**
+   * Prefix Vuetify composables:
+   * - when prefix set to `true` will use `useV`: `useVDate`.
+   * - when prefix is a string will use `use<prefix>`: `useVuetifyDate` with `prefix: 'Vuetify'`.
+   */
+  prefix?: true | string
+}
+
+export interface VuetifyDirectivesOptions {
+  /**
+   * Prefix Vuetify directives:
+   * - when prefix set to `true` will use `Vuetify` => `v-vuetify-<directive>: `v-vuetify-ripple`.
+   */
+  prefix?: true
+  exclude?: DirectiveName[]
+}
+
+export function VuetifyComposables (options: VuetifyComposablesOptions = {}) {
+  const { prefix } = options
+  const composableImports: [link: string, name: string][] = [
+    ['use-date', 'useDate'],
+    ['use-defaults', 'useDefaults'],
+    ['use-display', 'useDisplay'],
+    ['use-go-to', 'useGoTo'],
+    ['use-layout', 'useLayout'],
+    ['use-locale', 'useLocale'],
+    ['use-rtl', 'useRtl'],
+    ['use-theme', 'useTheme'],
+  ]
+  const imports = typeof prefix === 'string'
+    ? composableImports.map(([l, n]) => [l, n, n.replace('use', `use${prefix}`)])
+    : prefix
+      ? composableImports.map(([l, n]) => [l, n, n.replace('use', 'useV')])
+      : composableImports
+  return {
+    from: 'vuetify',
+    imports: imports.map(([link, name, renamed]) => ({
+      name,
+      as: renamed,
+      meta: { docsUrl: `https://vuetifyjs.com/en/api/${link}/` },
+    } satisfies PresetImport)),
+  } satisfies Preset
+}
+
+export function VuetifyOneComposables (options: VuetifyOneComposablesOptions = {}) {
```

Comment:
This shouldn't be part of vuetify, it's only for our sites.

Reply:
Ok